frontend/views/containers/access/LoginModal.vue

```diff
@@ -27,7 +27,7 @@ export default ({
   methods: {
     submit () {
       this.$refs.modal.close()
-      this.$router.push({ query: this.$route.query, path: this.$route.query.next ?? '/dashboard' }).catch(() => {})
+      this.$router.push({ query: this.$route.query, path: this.$route.query.next ?? '/' }).catch(() => {})
```

Comment:
Why was this changed?

Reply:
I think both should work, but `/` always has the same number of redirects. That's why I changed it here.